web-common/src/components/menu/wrappers/SimpleActionMenu.svelte

```diff
@@ -41,6 +41,7 @@ and the menu closes.
   <slot {handleClose} toggleMenu={toggleFloatingElement} {active} />
   <Menu
     slot="floating-element"
+    let:handleClose
```

Comment:
Why is this needed? Wont it mask the variable defined above on `WithTogglableFloatingElement`?